proposals/0387-cross-compilation-destinations.md

```diff
@@ -260,7 +373,7 @@ which then will be stored as configuration for this destination.
 After a destination is installed, users can refer to it via its identifier passed to the `--destination` option, e.g.
 
 ```
-swift build --destination ubuntu-jammy
+swift build --destination ubuntu_jammy
```

Comment:
Since jammy lists multiple runtime triples above, this won't work, unless the default is to build all of them.